substreams/ethereum-lido/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name = "ethereum-lido"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "ethereum_lido"
+crate-type = ["cdylib"]
+
+[dependencies]
+substreams = "0.5.22"
+substreams-ethereum = "0.9.9"
+prost = "0.11"
+substreams-helper = { git = "https://github.com/propeller-heads/tycho-protocol-sdk.git", rev = "b8aeaa3" }
+tycho-substreams = { git = "https://github.com/propeller-heads/tycho-protocol-sdk.git", rev = "52d5021" }
```

Let's try to use the latest versions here


```suggestion
substreams-helper = { git = "https://github.com/propeller-heads/tycho-protocol-sdk.git", tag = "0.5.0" }
tycho-substreams = "0.6.0"
```